runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1540,27 +1540,8 @@ void J9::ARM64::PrivateLinkage::performPostBinaryEncoding()
                    "Expecting the return type info instruction to be created");
 
    TR::ARM64ImmInstruction *linkageInfoWordInstruction = cg()->getReturnTypeInfoInstruction();
-   uint32_t linkageInfoWord = linkageInfoWordInstruction->getSourceImmediate();
-
-   intptr_t jittedMethodEntryAddress = reinterpret_cast<intptr_t>(getJittedMethodEntryPoint()->getBinaryEncoding());
-   intptr_t interpretedMethodEntryAddress = reinterpret_cast<intptr_t>(getInterpretedMethodEntryPoint()->getBinaryEncoding());
-
-   linkageInfoWord = (static_cast<uint32_t>(jittedMethodEntryAddress - interpretedMethodEntryAddress) << 16) | linkageInfoWord;
+   uint32_t linkageInfoWord = self()->initializeLinkageInfo(linkageInfoWordInstruction->getBinaryEncoding());
```

Comment:
I don't think this is correct.  `initializeLinkageInfo` is in the `CodeGenerator` class, not `Linkage`.

Reply:
Ah you're right; I didn't realize this was the private linkage class. Good catch.